tests/data/profile_stability/rhel8/stig.profile

```diff
@@ -247,6 +247,7 @@ selections:
 - sssd_offline_cred_expiration
 - sudo_remove_no_authenticate
 - sudo_remove_nopasswd
+- sudo_restrict_privilege_elevation_to_authorized
```

Comment:
please rebase this PR and add the same rule to the RHEL8 STIG with GUI profile, which is a new profile:

`tests/data/profile_stability/rhel8/stig_gui.profile`